.github/workflows/build.yml

```diff
@@ -52,5 +52,8 @@ jobs:
             exit 1
           fi
```

Comment:
Ahh I think it retagged our existing gotify/build images for 1.22.4 :/. This could break future plugin builds for this version.

Reply:
No I manually tagged it before renovate. It should be the same version on docker

Reply:
Well yes, but it still got pushed in https://github.com/gotify/build/actions/runs/11627886591/job/32382047849 
![image](https://github.com/user-attachments/assets/eb1a2bd6-d89c-44ad-8633-ff17bdc4fe10)

Reply:
I think it should be okay but still should have thought about this  :(... I pushed an empty commit on my gotify and it pulled the new tag succeeded: https://github.com/eternal-flame-AD/gotify-webhook-misskey/actions/runs/11635464745

Unless it was something using something just-introduced to debian it should be fine